pkg/apis/metrics/register.go

```diff
@@ -48,8 +48,12 @@ func addKnownTypes(scheme *runtime.Scheme) {
 	scheme.AddKnownTypes(SchemeGroupVersion,
 		&RawNode{},
 		&RawPod{},
+		&Node{},
+		&Pod{},
 	)
 }
 
 func (obj *RawNode) GetObjectKind() unversioned.ObjectKind { return &obj.TypeMeta }
 func (obj *RawPod) GetObjectKind() unversioned.ObjectKind  { return &obj.TypeMeta }
```

Comment:
(aside) We should delete these place holder values, since they're no longer used.